Input/Qualification-Ontogeny-UGT1A9.json

```diff
@@ -1,35 +1,30 @@
 {
-    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v2.1/OSP_Qualification_Plan_Schema.json",
+    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v3.0/OSP_Qualification_Plan_Schema.json",
     "Projects": [
         {
             "Id": "Propofol",
             "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Propofol-Model/master/Propofol.json",
```

Comment:
PAth must be replace with the reference to the released version (once Propofol is released).
Will create an issue for that.